extensions/cleanshotx/package.json

```
@@ -0,0 +1,140 @@
+{
+    "$schema": "https://www.raycast.com/schemas/extension.json",
+    "name": "cleanshotx",
+    "title": "Cleanshot X",
+    "description": "Capture and record your screen!",
+    "icon": "command-icon.png",
+    "author": "Aayush9029",
+    "license": "MIT",
+    "commands": [{
+            "name": "capture-fullscreen",
+            "title": "Capture Fullscreen",
+            "subtitle": "CleanShotX",
+            "description": "Fullscreen screenshot",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "capture-area",
+            "title": "Capture Area",
+            "subtitle": "CleanShotX",
+            "description": "Selection tool capture",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "capture-previous-area",
+            "title": "Capture Previous Area",
+            "subtitle": "CleanShotX",
+            "description": "Previously selected area",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "capture-window",
+            "title": "Capture Window",
+            "subtitle": "CleanShotX",
+            "description": "Window capture",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "self-timer",
+            "title": "Self Timer",
+            "subtitle": "CleanShotX",
+            "description": "Timer based screen capture",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "scrolling-capture",
+            "title": "Scrolling Capture",
+            "subtitle": "CleanShotX",
+            "description": "Scrolling capture mode",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "record-screen",
+            "title": "Record Screen",
+            "subtitle": "CleanShotX",
+            "description": "Record entire screen",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "text-recognition",
+            "title": "Text Recognition",
+            "subtitle": "CleanShotX",
+            "description": "Text recognition using OCR",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "annotate",
+            "title": "Annotate",
+            "subtitle": "CleanShotX",
+            "description": "Annotate mode",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "open-from-clipboard",
+            "title": "Open from clipboard",
+            "subtitle": "CleanShotX",
+            "description": "Open image from clipboard",
+            "mode": "no-view"
+        },
+
+        {
+            "name": "toggle-desktop-icons",
+            "title": "Toggle Desktop icons",
+            "subtitle": "CleanShotX",
+            "description": "Toggle desktop icons visibility",
```

```suggestion
            "description": "Toggle Desktop Icons Visibility",
```

Documentation: 
- [Extensions and Commands Naming](https://developers.raycast.com/basics/prepare-an-extension-for-store#extensions-and-commands-naming)
  - Extension and command titles should follow the Title Case convention